src/js/node/http2.ts

```diff
@@ -1846,14 +1848,15 @@ class Http2Stream extends Duplex {
           chunk,
           encoding,
           (this[bunHTTP2StreamStatus] & StreamState.EndedCalled) !== 0,
-          callback,
+          function () {
```

Comment:
this is always signal that we have backpressure on http2 which is not true, if we buffer something it will only going to call the callback when the socket is writable